diff --git a/node_modules/react-native-modalize/lib/index.js b/node_modules/react-native-modalize/lib/index.js
index 2e58fda..4efc5c6 100644
--- a/node_modules/react-native-modalize/lib/index.js
+++ b/node_modules/react-native-modalize/lib/index.js
@@ -635,7 +635,7 @@ onOpen, onOpened, onClose, onClosed, onBackButtonPress, onPositionChange, onOver
             styles_1.default.modalize__content,
             modalStyle,
             {
-                height: modalHeightValue,
+                height: modalHeightValue ? undefined : modalHeightValue,
                 maxHeight: endHeight,
                 transform: [
                     {
